.github/workflows/main.yml

```diff
@@ -20,14 +20,19 @@ jobs:
     name: Tests
     runs-on: ubuntu-latest
     steps:
+      - run: apt update
+      - run: apt install -y chromium-browser
       - uses: actions/checkout@v1
-      - name: Use Node.js 12
+      - name: Use Node.js 18
         uses: actions/setup-node@v1
         with:
-          node-version: 12
+          node-version: 18
```

Comment:
Two notes about the suggested Node v18 upgrade here:

  * Currently, Node v20 (`iron`) is the long-term-support (LTS) version of Node - so that might be a better choice here.
  * These dependency upgrades should potentially be isolated into a separate pull request (rationale: change one aspect of a project at a time, to identify as much possible breakage -- because then we can resolve that -- as possible).

I'll place this pull request into draft status until the dependencies are updated.